test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -8060,21 +8060,24 @@ BOOST_AUTO_TEST_CASE(create_memory_array_allocation_size)
 	// multiple of 32
 	char const* sourceCode = R"(
 		contract C {
-			function f() pure returns (uint d1, uint d2, uint d3) {
+			function f() pure returns (uint d1, uint d2, uint d3, bool ms) {
 				bytes memory b1 = new bytes(31);
 				bytes memory b2 = new bytes(32);
 				bytes memory b3 = new bytes(256);
 				bytes memory b4 = new bytes(31);
+				uint memsize;
 				assembly {
 					d1 := sub(b2, b1)
 					d2 := sub(b3, b2)
 					d3 := sub(b4, b3)
+					memsize := msize()
 				}
+				ms = memsize <= (480 + 0x80);
```

Comment:
Can we check for equality here instead?

Reply:
If we do, the test will break as soon as some initialization routine needs more memory.

Reply:
Maybe that's a good thing?

Reply:
Rigth, it is as long as it is no big deal to update the expectations. This is where you come in ;)